tests/ftests/065-sudo-systemd_cgclassify-v1.py

```diff
@@ -0,0 +1,196 @@
+                    '/sys/fs/cgroup/{}/{}/{}/'.format(SYSTEMD_CGNAME, CONTROLLER, SLICE, SCOPE)
+                )
+        return result, cause
+
+    # create and check if the cgroup was created under the controller sub-tree
+    if not Systemd.create_and_validate(config, CONTROLLER, OTHER_CGNAME, ignore_systemd=True):
+        result = consts.TEST_FAILED
+        cause = (
+                    'Failed to create cgroup {} under '
+                    '/sys/fs/cgroup/{}/'.format(OTHER_CGNAME, CONTROLLER)
+                )
+
+    return result, cause
+
+
+def create_process_get_pid(config, CGNAME, SLICE='', ignore_systemd=False):
+    result = consts.TEST_PASSED
+    cause = None
+
+    config.process.create_process_in_cgroup(
+                                                config, CONTROLLER, CGNAME,
+                                                ignore_systemd=ignore_systemd
+                                            )
+
+    pids = Cgroup.get_pids_in_cgroup(config, os.path.join(SLICE, CGNAME), CONTROLLER)
+    if pids is None:
+        result = consts.TEST_FAILED
+        cause = 'No processes were found in cgroup {}'.format(CGNAME)
+
+    return pids, result, cause
+
+
+def terminate_process(config, pids):
+    if pids:
+        for p in pids.splitlines():
+            Run.run(['sudo', 'kill', '-9', p])
+
+
+def test(config):
+    global SYSTEMD_PIDS, OTHER_PIDS
+
+    result = consts.TEST_PASSED
+    cause = None
+
+    # Test cgclassify, that creates a process and then uses cgclassify
+    # to migrate the task the cgroup.
+    SYSTEMD_PIDS, result, cause = create_process_get_pid(
+                                                            config, SYSTEMD_CGNAME,
+                                                            os.path.join(SLICE, SCOPE)
+                                                        )
+    if result == consts.TEST_FAILED:
+        return result, cause
+
+    OTHER_PIDS, result, tmp_cause = create_process_get_pid(
+                                                                config, OTHER_CGNAME,
+                                                                ignore_systemd=True
+                                                          )
+    if result == consts.TEST_FAILED:
+        return result, cause
+
+    # classify the task from the non-systemd scope cgroup to systemd scope
+    # cgroup (should fail due to wrong path).
+    try:
+        Cgroup.classify(config, CONTROLLER, SYSTEMD_CGNAME, OTHER_PIDS, ignore_systemd=True)
+    except RunError as re:
+        err_str = 'Error changing group of pid {}: Cgroup does not exist'.format(OTHER_PIDS)
+        if re.stderr != err_str:
+            raise re
+    else:
+        result = consts.TEST_FAILED
+        cause = 'Changing group of pid {} erroneously succeeded'.format(OTHER_PIDS)
+
+    # classify the task from the non-systemd scope cgroup to systemd scope
+    # cgroup (should fail due to wrong path).
```

Comment:
nitpick - I think this comment isn't right.

Reply:
Does the following re-phrasing make sense?
```      
# This should fail because of trying to migrate a task to SYSTEMD_CGNAME
# under the wrong path from OTHER_CGNAME.
```

Reply:
It's a copy-paste of the previous comment.  Would this be correct?
```
# classify the task from the systemd scope cgroup to the non-systemd scope
# cgroup (should fail due to wrong path).
```

Reply:
Agreed, re-reading it, it's not obvious. Does the more verbose version make sense?
```
    # classify a task from the non-systemd scope cgroup (OTHER_CGNAME) to systemd                   
    # scope cgroup (SYSTEMD_CGNAME).  Migration should fail due to the incorrect                    
    # destination cgroup path that gets constructed, without the systemd slice/scope                
    # when ignore_systemd is set) 
```